docs/environment/database-planetscale.md

```diff
@@ -133,7 +133,7 @@ doctrine:
         url: '%env(resolve:DATABASE_URL)%'
         options:
             # Connect to the database via SSL
-            !php/const:PDO::MYSQL_ATTR_SSL_CA: /opt/bref/ssl/cert.pem
+            !php/const PDO::MYSQL_ATTR_SSL_CA: /opt/bref/ssl/cert.pem
```

Comment:
Wait don't we need quotes to escape that? Like this:

```suggestion
            '!php/const PDO::MYSQL_ATTR_SSL_CA': /opt/bref/ssl/cert.pem
```

Have you tested this and it's working correctly for you?

Reply:
<img width="685" alt="Screenshot 2023-04-24 at 10 46 21" src="https://user-images.githubusercontent.com/668391/233946192-1c22798d-0d15-440e-b5a5-39281bdbc75a.png">

Works unquoted indeed.

Reply:
OK let's do that then, thanks!

Reply:
I think it's safe enough. I found the official docs also use the unquoted notation here:

https://symfony.com/doc/current/configuration.html#configuration-parameters

```
# config/services.yaml
parameters:
    # PHP constants as parameter values
    app.some_constant: !php/const GLOBAL_CONSTANT
    app.another_constant: !php/const App\Entity\BlogPost::MAX_ITEMS
```